src/harness/vfs.ts

```diff
@@ -404,7 +409,21 @@ namespace vfs {
         }
 
         /**
-         * Get file status.
+         * Change file access times
+         *
+         * NOTE: do not rename this method as it is intended to align with the same named export of the "fs" module.
+         */
+        public utimesSync(path: string, atime: Date, mtime: Date) {
+            const entry = this._walk(this._resolve(path));
+            if (!entry || !entry.node) {
+                throw createIOError("ENOENT");
+            }
+            entry.node.atimeMs = +atime;
+            entry.node.mtimeMs = +mtime;

```

Comment:
You should also set `entry.node.ctimeMs = this.time()` to set the status change time when you update `atimeMs` or `mtimeMs`.